maven-archetype-quickstart/src/main/resources-filtered/archetype-resources/pom.xml

```diff
@@ -6,6 +6,7 @@
 #else
     <maven.compiler.source>${javaCompilerVersionLocal}</maven.compiler.source>
     <maven.compiler.target>${javaCompilerVersionLocal}</maven.compiler.target>
+    <!--maven.compiler.target>11</maven.compiler.target-->
```

Comment:
I think this comment belongst to: https://github.com/apache/maven-archetypes/blob/master/maven-archetype-quickstart/src/main/resources-filtered/archetype-resources/pom.xml#L49

Reply:
#27 merged, I'll rebase: this will ease understanding the scope of this precise PR

Reply:
this comment is best placed here because it is activated only when javaCompilerVersionLocal <= 8, based on effective value at runtime